test/test_parmed_rdkit.py

```diff
@@ -35,3 +35,13 @@ def test_load_smiles(self):
         # coordinates = True (default)
         parm = pmd.rdkit.from_smiles(smiles)
         np.testing.assert_allclose(parm.coordinates[0], [-1.072,  0.829 ,  0.108])
+
+    def test_load_smiles_explicit_hydrogen(self):
```

Comment:
The other tests would need `hydrogens=False` added to the `from_smiles` calls to continue passing with the provided recommendation.

Reply:
Sounds great! I’ll get those necessary changes in, in the morning.